tests/scripts/seviri.WV-Upper.unprojected_image.sh

```diff
@@ -16,23 +16,33 @@
-             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000008___-202004040800-C_ \
+# run_procflow $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-_________-EPI______-202004040800-__ \
+#              $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-_________-PRO______-202004040800-__ \
+#              $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000001___-202004040800-C_ \
+#              $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000002___-202004040800-C_ \
+#              $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000003___-202004040800-C_ \
+#              $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000004___-202004040800-C_ \
+#              $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000005___-202004040800-C_ \
+#              $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000006___-202004040800-C_ \
+#              $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000007___-202004040800-C_ \
+#              $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000008___-202004040800-C_ \
+run_procflow $GEOIPS_TESTDATA_DIR/test_data_seviri/data/H-000-MSG2__-MSG2_IODC___-_________-EPI______-202312110800-__ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/H-000-MSG2__-MSG2_IODC___-_________-PRO______-202312110800-__ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/H-000-MSG2__-MSG2_IODC___-WV_062___-000001___-202312110800-C_ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/H-000-MSG2__-MSG2_IODC___-WV_062___-000002___-202312110800-C_ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/H-000-MSG2__-MSG2_IODC___-WV_062___-000003___-202312110800-C_ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/H-000-MSG2__-MSG2_IODC___-WV_062___-000004___-202312110800-C_ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/H-000-MSG2__-MSG2_IODC___-WV_062___-000005___-202312110800-C_ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/H-000-MSG2__-MSG2_IODC___-WV_062___-000006___-202312110800-C_ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/H-000-MSG2__-MSG2_IODC___-WV_062___-000007___-202312110800-C_ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/H-000-MSG2__-MSG2_IODC___-WV_062___-000008___-202312110800-C_ \
```

Comment:
@jsolbrig  We could just switch to this test case (uncomment the "compare_path" and update the outputs to match this test case rather than the other one).

Reply:
I don't have that particular dataset.  We should coordinate and grab the same recent datafile, and update the test script / test output?  In any case, we can't merge this test script as is.

Reply:
Yeah, we should coordinate on this. Would it work to just give you this dataset or would it be better to have you find something to test with?

Reply:
I have the used dataset with me and can send it now if we wish to proceed with it

Reply:
@evrose54 either revert this back, or put together a test data repo with this test dataset and push to CIRA NEXCLOUD  (and update the test output accordingly) ?

Reply:
@evan will create a test data repo, tar it, add it to nextcloud and provide it to NRL.

Reply:
Here is the test dataset for the updated script. The data files above reference the old script, but posting the link here nonetheless. https://io2.cira.colostate.edu/s/XxP4r8MSPesExQr